react/src/components/molecules/EmergencyHeader/style.scss

```diff
@@ -0,0 +1,63 @@
+@import 'global';
+@import '00-base/mixins';
+@import '02-molecules/emergency-header';
+
+@mixin ma-emergency-header ($color: $c-highlight) {
```

Comment:
this should be moved into the assets file for this molecule that is imported above (lines 5 - 63)

Reply:
maybe create a mixin file for this